packetbeat/docs/fields.asciidoc

```diff
@@ -10058,7 +10105,7 @@ type: array
 
 --
 
-*`tls.client_certificate_chain`*::
+*`tls.detailed.client_certificate_chain`*::
```

Comment:
how is this different than `tls.client.certificate_chain`? I see the code referencing both.

Reply:
This is an array of objects, same fields as `tls.detailed.client.certificate`, while the ECS counterpart just stores the base64 PEM.